spec/lib/appsignal_spec.rb

```diff
@@ -92,18 +92,23 @@
 
       context "when allocation tracking and gc instrumentation have been enabled" do
         before do
+          GC::Profiler.stub(:enable)
```

Comment:
using the `allow(GC::Profiler).to receive(:enable)` syntax matches more with the `expect(GC::Profiler).to receive(:method_name)` you use elsewhere. It does the same thing